<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Composite_Index</class>
 <widget class="QWidget" name="Composite_Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incidence</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Layer</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsMapLayerComboBox" name="cbx_aggregation_layer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Indicator field</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QgsFieldComboBox" name="cbx_indicator_field">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Composite Index</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLineEdit" name="le_output_filepath">
         <property name="placeholderText">
          <string>Save to temporary file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_browse">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="radioButton_vector_positive">
       <property name="text">
        <string>Higher the value of the indicator better the conditions for the population</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="radioButton_vector_negative">
       <property name="text">
        <string>Higher the value of the indicator worse the conditions for the population</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_2</string>
       </attribute>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="le_new_column">
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="placeholderText">
        <string>UHN</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QListWidget" name="cbx_list_indicators"/>
     </item>
     <item row="8" column="1">
      <widget class="QCommandLinkButton" name="command_link_button">
       <property name="text">
        <string>Add to Index</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>For removing and Indicator Click Twice in the one you want to remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="symbology">
     <property name="title">
      <string>Add a symbology</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="collapsed" stdset="0">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Low Priority</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsColorButtonV2" name="color_low_value">
          <property name="color" stdset="0">
           <color>
            <red>50</red>
            <green>164</green>
            <blue>46</blue>
           </color>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>High Priority</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QgsColorButtonV2" name="color_high_value">
          <property name="color" stdset="0">
           <color>
            <red>202</red>
            <green>33</green>
            <blue>36</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Classes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_classes">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbx_mode"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box_ok">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgis.gui</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButtonV2</class>
   <extends>QToolButton</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbx_aggregation_layer</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>cbx_indicator_field</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
